docs/manual/forms/basics.rst

```diff
@@ -78,6 +78,14 @@ In dit tabblad worden de algemene formulier gegevens weergegeven.
   bijvoorbeeld alleen informatie of verwijzingen naar andere formulieren. In dat
   geval vinkt u deze optie uit.
 
+.. note::
+
+  Als de formuliernaam lange woorden bevat, optioneel koppeltekens kunnen ingevoerd worden (`soft hyphens`_) om door
+  te geven waar het woord kan worden afgebroken om naar de volgende regel te gaan. Op Windows, kunt u soft hyphens in
+  de 'Character map' vinden en kopiëren/plakken naar Open Forms velden.
```

Comment:
```suggestion
.. note::

  Als de formuliernaam lange woorden bevat, kunt u optionele koppeltekens (`soft hyphens`_) invoeren om aan te geven waar het woord mag afgebroken worden naar de volgende regel. Op Windows vindt u deze "soft hyphens" in
  het 'Speciale tekens'-programma  en kunt u ze kopiëren/plakken naar Open Forms-velden.
```
"kunnen ingevoerd worden" is passive form and is generally best avoided in user texts (if what they taught me in school is still accurate).